### PR TITLE
Add support to python 3.10

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-    py{36,37,38,39,py3}-dj{22,30,31,32}-{unittest,pytest}
-    py{38,39}-djmain-{unittest,pytest}
+    py{36,37,38,39,310,py3}-dj{22,30,31,32}-{unittest,pytest}
```
Comment (smithdc1) [inline tox.ini:3]: Django versions 2.2 through to 3.1 will not support Python 3.10. (Django 3.2 will in it's next release in early November)

File: setup.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ def find_version(*file_paths):
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.0',
         'Framework :: Django :: 3.1',
+        'Framework :: Django :: 3.2',
```
Comment (smithdc1) [inline setup.py:40]: I wonder if it is worth widening this PR to also add Django 4.0 support (add to test suite as well).

File: tox.ini [hunk 3]
```diff
@@ -1,7 +1,8 @@
 [tox]
 envlist =
     py{36,37,38,39,py3}-dj{22,30,31,32}-{unittest,pytest}
-    py{38,39}-djmain-{unittest,pytest}
+    py{38,39,310}-djmain-{unittest,pytest}
+    py{310}-dj{32,40}-{unittest,pytest}
```
Comment (smithdc1) [inline tox.ini:5]: 4.0 is also supported by 3.8,3.9 so maybe 🤔

```suggestion
    py{38,39,310}-dj{40,main}-{unittest,pytest}
    py310-dj32-{unittest,pytest}
```

**Review by smithdc1 — COMMENTED** [review]
Thanks for the updates!

**Comment by jairhenrique** [comment]
@smithdc1 

```bash
>> tox -l
py36-dj22-unittest
py36-dj22-pytest
py36-dj30-unittest
py36-dj30-pytest
py36-dj31-unittest
py36-dj31-pytest
py36-dj32-unittest
py36-dj32-pytest
py37-dj22-unittest
py37-dj22-pytest
py37-dj30-unittest
py37-dj30-pytest
py37-dj31-unittest
py37-dj31-pytest
py37-dj32-unittest
py37-dj32-pytest
py38-dj22-unittest
py38-dj22-pytest
py38-dj30-unittest
py38-dj30-pytest
py38-dj31-unittest
py38-dj31-pytest
py38-dj32-unittest
py38-dj32-pytest
py39-dj22-unittest
py39-dj22-pytest
py39-dj30-unittest
py39-dj30-pytest
py39-dj31-unittest
py39-dj31-pytest
py39-dj32-unittest
py39-dj32-pytest
pypy3-dj22-unittest
pypy3-dj22-pytest
pypy3-dj30-unittest
pypy3-dj30-pytest
pypy3-dj31-unittest
pypy3-dj31-pytest
pypy3-dj32-unittest
pypy3-dj32-pytest
py38-dj40-unittest
py38-dj40-pytest
py38-djmain-unittest
py38-djmain-pytest
py39-dj40-unittest
py39-dj40-pytest
py39-djmain-unittest
py39-djmain-pytest
py310-dj40-unittest
py310-dj40-pytest
py310-djmain-unittest
py310-djmain-pytest
py310-dj32-unittest
py310-dj32-pytest

**Pull request merged**